templates/web/base/waste/garden/renew.html

```diff
@@ -1,7 +1,129 @@
-[% BLOCK 'content' %]
+[% extra_js = [
+    version('/js/waste.js')
+] ~%]
+[% INCLUDE 'waste/header.html' %]
+
+[% PROCESS 'govuk/fields.html' %]
+[% PROCESS title title=form.title %]
+[% PROCESS errors %]
+
+<div class="waste-service-hint-banner">
+    <div class="waste-service-hint-banner__image">
+        <img src="/cobrands/bromley/images/container-images/garden-waste-bin.png" alt="">
+    </div>
+    <div class="waste-service-hint__content">
+        <p>
+            Our 240 litre wheelie bins have grey sides and a brown top, and hold the
+            equivalent of 5 or 6 bags of green garden waste.
+        </p>
+    </div>
+</div>
+
   <p>
   This is the text to explain how renewals work.
   </p>
-[% END %]
 
-[% INCLUDE 'waste/garden/property_page.html' %]
+<form method="post">
+  <fieldset class="govuk-fieldset">
+    <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+      <h2 class="govuk-fieldset__heading">How many bins do you need us to collect?</h2>
+    </legend>
+
+  <!-- [% PROCESS form override_fields=['current_bins'] %] -->
```

Comment:
It looks like this has been c'n'p from the subscription templates rather than refactored if they're basically the same? Though, why is the current bins number editable, if there's a separate additional bins box, for a renewal? It seems rather confusing as is. If a renewal only lets you keep same number or more, then I'd make current bins uneditable; if it's meant to also let you reduce the number of bins, then I'd only have the one box with number of bins required, because otherwise it reads like we'd want "2" in already on property and "-1" in additional bins required?

Smaller bug - if you leave either box blank, the total becomes `NaN`.